build.gradle

```diff
@@ -54,7 +54,7 @@ ext {
   terracottaPlatformVersion = '5.2.1'
   managementVersion = terracottaPlatformVersion
   terracottaApisVersion = '1.2.0'
-  terracottaCoreVersion = '5.2.1'
+  terracottaCoreVersion = '5.2.2'
```

Comment:
This cannot go in alone, we need the platform version bump as well. So waiting on terracotta-oss/terracotta-platform#301.